Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs

```diff
@@ -523,13 +523,14 @@ class CondaEnvironmentManagerUI : ICondaEnvironmentManagerUI {
 
             var paths = GetPathEntries(view);
             var pathCmd = string.IsNullOrEmpty(paths) ? "" : string.Format("set PATH={0};%PATH% & ", paths);
-            var psi = new ProcessStartInfo("cmd.exe");
-            psi.Arguments = string.Join(" ", new[] {
+            var psi = new ProcessStartInfo(Path.Combine(Environment.SystemDirectory, "cmd.exe")) {
+                Arguments = string.Join(" ", new[] {
                 "/S",
                 "/K",
                 pathCmd + string.Format("title {0} environment", view.Description)
-            }.Select(ProcessOutput.QuoteSingleArgument));
-            psi.WorkingDirectory = view.PrefixPath;
+            }.Select(ProcessOutput.QuoteSingleArgument)),
+                WorkingDirectory = view.PrefixPath
+            };
```

Comment:
This is hard to read with this indentation. Arguments and WorkingDirectory should be the only 2 lines at that indent level.

Reply:
Updated. Not sure if this is exactly what you wanted, but I feel that it's much easier to read.